src/app/objects/resume.ts

```diff
@@ -0,0 +1,57 @@
+import { template } from "./template";
+
+export interface resume {
+  template: template,
+  header: header,
+  workExps: workExp[],
+  educations: education[],
+  certificates?: string[],
+  skills: string[],
+  summary: string
+}
+
+export interface header {
+  id: number;
```

Comment:
Can Id be string instead? Mongo auto generates an Id called ObjectId which is globally unique and is a string once converted

Reply:
Also why do some interfaces have id while others don't?

Reply:
Actually I think we just need one id for `resume` interface? Other interfaces are just subsections of `resume`